ttnn/cpp/pybind11/pytensor.cpp

```diff
@@ -1512,6 +1512,15 @@ void pytensor_module(py::module &m_tensor) {
 
                 volume = tt_tensor.volume()
 
+        )doc")
+        .def(
+            "logical_volume", [](const Tensor &self) { return self.get_logical_volume(); }, R"doc(
```

Comment:
Discussed offline, agreed that long trm this must be "volume" and the existing function must be renamed to "physical_volume" or smth like that, in alignment with "shape" returning the logical shape

Reply:
marked with todos